mods/ra/maps/allies-03a/map.yaml

```diff
@@ -1546,6 +1546,15 @@ Rules:
 	DOMF:
 		Buildable:
 			Prerequisites: ~disabled
+	ATEF:
+		Buildable:
+			Prerequisites: ~disabled
+	MSLF:
+		Buildable:
+			Prerequisites: ~disabled
+	PDOF:
+		Buildable:
+			Prerequisites: ~disabled
```

Comment:
You can remove those (and the older ones above) as you can't get an allied construction yard in allies03a.